server/test/graql/parser/ParserTest.java

```diff
@@ -909,99 +923,6 @@ public void testParseManyMatchInsertWithoutStackOverflow() {
         assertEquals(Collections.nCopies(numQueries, matchInsert), queries);
     }
 
-    @Test
```

Comment:
Because we no longer support parsing a continuous stream of input reader. We don't need this functionality internally, and it should be the responsibility of the user.